web/api/views.py

```
@@ -24,6 +24,7 @@
 from recon_note.models import *
 from reconPoint.celery import app
 from reconPoint.common_func import *
+from reconPoint.database_utils import *
```

Ruff

`from reconPoint.database_utils import *` used; unable to detect undefined names

(F403)